pages/releases/4.3.html

```diff
@@ -0,0 +1,2109 @@
+									data-read-more="https://github.com/godotengine/godot/pull/87939">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #gdscript-is-not-operator -->
+
+				<div id="gdscript-built-in-methods-callable" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#gdscript-built-in-methods-callable">
+										Built-in methods and utility functions usable as
+										<code>
+											<span class="basetype">Callable</span>
+										</code>
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										You have an array and you want to print each element without creating
+										a for loop?
+									</p>
+									<p>
+										By making built-in methods and utility functions usable as
+										<code>Callable</code> makes it possible to call variadic functions such as
+										<code class="highlight">
+											<span class="gdscript-globalfunction">print</span><span class="symbol">()</span>
+										</code>
+										with
+										<code class="highlight">
+											<span class="basetype">Callable</span><span class="symbol">.</span><span class="function">callv</span><span class="symbol">()</span>
+										</code>
+										easily, without the need of boilerplate code.
+									</p>
+									<p>
+										You can now call this:<br>
+										<code class="highlight">
+											print<span class="symbol">.</span><span class="function">callv</span><span class="symbol">(</span>my_array<span class="symbol">)</span>
+										</code>
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="dalexeev"
+									data-read-more="https://github.com/godotengine/godot/pull/82264">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #gdscript-built-in-methods-callable -->
+
+				<div id="gdscript-export-storage-property-hint" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#gdscript-export-storage-property-hint">
+										<code>
+											<span class="gdscript-annotation">@export_storage</span>
+										</code>
+										property hint
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										Addon developers, you can now use
+										<code class="highlight">
+											<span class="gdscript-annotation">@export_storage</span>
+										</code>
+										to store hidden
+										values in a scene.
+									</p>
+									<p>
+										Useful if you need to store data on a node or a resource that need to be
+										accessed by the addon, but that isn't meant to be edited by the user in the
+										inspector.
+									</p>
+								</div>
```

Comment:
```suggestion
								<h4 class="c-title">
									<a href="#gdscript-export-storage-property-hint">
										<code>
											<span class="gdscript-annotation">@export_storage</span>
										</code>
										property annotation
									</a>
								</h4>
								<div class="c-content">
									<p>
										This feature is for the plugin developers amongst you.
									</p>
									<p>
										Using the new
										<code class="highlight">
											<span class="gdscript-annotation">@export_storage</span>
										</code>
										 annotation allows you to store hidden values in a scene.
									</p>
									<p>
										A common use case for this could be storing addon information in a node and preventing users from (accidentally) editing it in the inspector.
									</p>
								</div>
```